### DISCUSSION TITLE
v1.0.2

### DISCUSSION THREAD

**Discussion by BlackthornYugen** [mongo-express/mongo-express#1306] [Announcements]
## What's Changed
* build(deps-dev): bump mongodb-memory-server from 8.9.3 to 8.9.4 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1029
* build(deps-dev): bump babel-loader from 8.2.5 to 9.0.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1027
* build(deps-dev): bump mongodb-memory-server from 8.9.4 to 8.9.5 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1030
* build(deps-dev): bump babel-loader from 9.0.0 to 9.0.1 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1031
* build(deps-dev): bump babel-loader from 9.0.1 to 9.1.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1032
* build(deps-dev): bump eslint from 8.26.0 to 8.27.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1033
* build(deps-dev): bump @babel/core from 7.19.6 to 7.20.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1034
* build(deps-dev): bump @babel/preset-env from 7.19.4 to 7.20.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1035
* build(deps-dev): bump chai from 4.3.6 to 4.3.7 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1036
* build(deps-dev): bump mongodb-memory-server from 8.9.5 to 8.10.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1039
* build(deps-dev): bump webpack from 5.74.0 to 5.75.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1040
* build(deps-dev): bump cypress from 10.11.0 to 11.0.1 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1041
* build(deps-dev): bump node-html-parser from 6.1.1 to 6.1.3 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1042
* build(deps-dev): bump node-html-parser from 6.1.3 to 6.1.4 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1043
* Create LICENSE.md respecting MIT in Readme by @shakaran in https://github.com/mongo-express/mongo-express/pull/1044
* Clean and lint license by @rtritto in https://github.com/mongo-express/mongo-express/pull/1045
* build(deps-dev): bump cypress from 11.0.1 to 11.1.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1046
* skip installation of dev packages for docker image by @bruegth in https://github.com/mongo-express/mongo-express/pull/1047
* add build stage to build with docker by @bruegth in https://github.com/mongo-express/mongo-express/pull/1050
* build(deps): bump mongodb from 4.10.0 to 4.12.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1049
* build(deps-dev): bump @babel/core from 7.20.2 to 7.20.5 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1057
* build(deps-dev): bump concurrently from 7.5.0 to 7.6.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1058
* build(deps-dev): bump eslint from 8.27.0 to 8.28.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1060
* build(deps-dev): bump cypress from 11.1.0 to 11.2.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1061
* build(deps): bump mongodb from 4.12.0 to 4.12.1 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1062
* build(deps-dev): bump mongodb-memory-server from 8.10.0 to 8.10.1 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1063
* build(deps-dev): bump eslint from 8.28.0 to 8.29.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1064
* build(deps-dev): bump supertest from 6.3.1 to 6.3.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1065
* Use replace document on update by @rtritto in https://github.com/mongo-express/mongo-express/pull/1051
* Replace custom pager and pagination with paginator-bottom by @rtritto in https://github.com/mongo-express/mongo-express/pull/1053
* Remove dirty class of read_only by @rtritto in https://github.com/mongo-express/mongo-express/pull/1054
* build(deps-dev): bump eslint-plugin-unicorn from 44.0.2 to 45.0.1 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1059
* build(deps-dev): bump webpack-cli from 4.10.0 to 5.0.1 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1066
* build(deps-dev): bump supertest from 6.3.2 to 6.3.3 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1068
* build(deps-dev): bump eslint-plugin-unicorn from 45.0.1 to 45.0.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1070
* build(deps-dev): bump mongodb-memory-server from 8.10.1 to 8.10.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1071
* build(deps-dev): bump mocha from 10.1.0 to 10.2.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1072
* build(deps-dev): bump jquery from 3.6.1 to 3.6.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1074
* build(deps-dev): bump eslint from 8.29.0 to 8.30.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1078
* build(deps): bump mongodb from 4.12.1 to 4.13.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1079
* Fix uuid v3 by @rtritto in https://github.com/mongo-express/mongo-express/pull/1085
* build(deps-dev): bump jquery from 3.6.2 to 3.6.3 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1083
* build(deps-dev): bump @babel/core from 7.20.5 to 7.20.7 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1087
* build(deps-dev): bump cypress from 11.2.0 to 12.2.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1084
* Add Mongo Express Docker Extension under README by @ajeetraina in https://github.com/mongo-express/mongo-express/pull/1080
* Improve find and aggregate by @rtritto in https://github.com/mongo-express/mongo-express/pull/1088
* Improve `viewCollection` with Promise.all by @rtritto in https://github.com/mongo-express/mongo-express/pull/1090
* build(deps-dev): bump cypress from 12.2.0 to 12.4.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1111
* build(deps): bump cookiejar from 2.1.3 to 2.1.4 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1109
* build(deps): bump json5 from 1.0.1 to 1.0.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1101
* build(deps-dev): bump babel-loader from 9.1.0 to 9.1.2 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1098
* Remove dirty accordian code by @rtritto in https://github.com/mongo-express/mongo-express/pull/1092
* build(deps-dev): bump eslint-plugin-import from 2.26.0 to 2.27.5 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1112
* build(deps-dev): bump eslint from 8.30.0 to 8.32.0 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1105
* build(deps-dev): bump @babel/core from 7.20.7 to 7.20.12 by @dependabot in https://github.com/mongo-express/mongo-express/pull/1099
* Fix delete all documents by @rtritto in https://github.com/mongo-express/mongo-express/pull/1116
* Fix success message in delete many documents by @rtritto in https://github.com/mongo-express/mongo-express/pull/1117
* Update deps 230203 by @rtritto in https://github.com/mongo-express/mongo-express/pull/1124
* Print mongo-express version on startup by @akbhuker in https://github.com/mongo-express/mongo-express/pull/1125

## New Contributors
* @bruegth made their first contribution in https://github.com/mongo-express/mongo-express/pull/1047
* @ajeetraina made their first contribution in https://github.com/mongo-express/mongo-express/pull/1080
* @akbhuker made their first contribution in https://github.com/mongo-express/mongo-express/pull/1125

**Full Changelog**: https://github.com/mongo-express/mongo-express/compare/v1.0.1...v1.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/mongo-express/mongo-express/releases/tag/v1.0.2'>v1.0.2</a>.</em>

**Comment by bingneef** [comment]
Hello!
Great work! I was wondering if this release can also be made available as a docker image. I seems the latest version on [dockerhub](https://hub.docker.com/_/mongo-express) is 1.0.0.
Thank you!

**Reply by BlackthornYugen** [reply]
I keep failing to get the docker build to run. We changed to yarn in 1.0.1 and it was causing some issues with our Docker build. I should have some time to look into it this coming weekend.
